### PR TITLE
Bolo do caco recipe added 🇵🇹

### PR DESCRIPTION
**Pull request by joaoofreitas**
Hello, added a really good bread recipe from the island I live in Portugal. Hope you like it!

### CODE REVIEW

File: src/bolo-do-caco.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Bolo do Caco _(Caco Bread)_
+
```
Comment (Knarkzel) [inline src/bolo-do-caco.md:2]: Missing image.

File: src/bolo-do-caco.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- ⅔ cup water (warm at 75F / 35C)
+- 3 tablespoons active dry - yeast
+- 3 teaspoons salt
+
+__Metric__
+- 750 g sweet potato
+- 1 kg all-purpose flour
+- 150 ml water (warm at 75F / 35C)
+- 3 tablespoons active dry yeast
+- 3 teaspoons salt
```
Comment (Knarkzel) [inline src/bolo-do-caco.md:25]: Interesting way to format imperial and metric. I think this is fine.

Comment (Knarkzel) [inline src/bolo-do-caco.md:25]: Don't include salt and pepper and other ubiquitous things in the ingredients list.

Comment (joaoofreitas) [reply src/bolo-do-caco.md:25]: Hello, do you need me to remove it?

**Pull request merged**